frontend/src/components/licences/LicenceHeader.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-container fluid class="px-0">
+    <h4 class="mb-2">Licence {{ licence?.licence }}</h4>
+    <v-row no-gutters>
+      <v-col cols="12" lg="6">
+        <v-row no-gutters class="mr-2 my-2">
+          <v-col cols="12" sm="6" class="pr-2 mt-1">
+            <AppLabel>CCOF Facility ID:</AppLabel>
+          </v-col>
+          <v-col cols="12" sm="6" class="mt-1">{{ licence?.ccofFacilityId ? licence?.ccofFacilityId : '- - - -' }}</v-col>
```

Comment:
I might create a local constant for '- - - -' since it's used quite often. Remember constants are best defined in created() since they don't need to be reactive.

Reply:
it is updated